dashboard/app/models/lesson.rb

```diff
@@ -765,7 +764,11 @@ def copy_to_script(destination_script)
           copied_activity_section.key = SecureRandom.uuid
           copied_activity_section.lesson_activity_id = copied_lesson_activity.id
           copied_activity_section.save!
-          sl_data = original_activity_section.script_levels.map.with_index(1) {|l, pos| JSON.parse({assessment: l.assessment, bonus: l.bonus, challenge: l.challenge, levels: l.levels, activitySectionPosition: pos}.to_json)}
+          sl_data = original_activity_section.script_levels.map.with_index(1) do |original_script_level, pos|
+            original_active_level = original_script_level.oldest_active_level
```

Comment:
it could be worth noting in comments here that this removes any inactive variants as part of copying.